src/scenes/Locations/LocationForm/FundingAccount/CreateFundingAccount.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Except } from 'type-fest';
+import { CreateFundingAccountInput } from '../../../../api';
+import {
+  DialogForm,
+  DialogFormProps,
+} from '../../../../components/Dialog/DialogForm';
+import {
+  NumberField,
+  SubmitError,
+  TextField,
+} from '../../../../components/form';
+import { FundingAccountLookupItem } from '../../../../components/form/Lookup/FundingAccount';
+import { useCreateFundingAccountMutation } from './CreateFundingAccount.generated';
+
+export type CreateFundingAccountProps = Except<
+  DialogFormProps<CreateFundingAccountInput, FundingAccountLookupItem>,
+  'onSubmit'
+>;
+
+export const CreateFundingAccount = (props: CreateFundingAccountProps) => {
+  const [createFundingAccount] = useCreateFundingAccountMutation();
+  return (
+    <DialogForm
+      {...props}
+      onSubmit={async (input) => {
+        const { data } = await createFundingAccount({
+          variables: { input },
+        });
+
+        return data!.createFundingAccount.fundingAccount;
+      }}
+      title="Create Funding Account"
+      fieldsPrefix="fundingAccount"
+    >
+      <SubmitError />
+      <TextField
+        name="name"
+        label="Account Name"
+        placeholder="Enter account name"
+        required
+      />
+      <NumberField
+        name="accountNumber"
+        label="Account Number"
+        placeholder="Enter account number"
+        formatInput={(numString) => numString.replace(',', '')}
```

Comment:
Still has the bug where if you click away on the field the commas come back, will look into it separately

Reply:
Let's not even worry about it. Production data has 7 funding accounts. All have "account numbers" of 0-7. There's no need to try and handle thousands separator.

Reply:
fixed with reppaceAll